detox/package.json

```diff
@@ -81,10 +81,17 @@
     "yargs-unparser": "^2.0.0"
   },
   "peerDependencies": {
-    "jest-circus": "26.0.x - 26.4.x || >=26.5.2",
-    "jest-environment-node": ">=25.0.0",
+    "jest": "26.0.x - 26.4.x || ^26.5.2 || 27.x.x || 28.x.x",
     "mocha": ">=6.0.0"
   },
+  "peerDependenciesMeta": {
+    "jest": {
+      "optional": true
+    },
+    "mocha": {
+      "optional": true
+    }
+  },
```

Comment:
Doesn't that effectively allows both to be absent from `node_modules/`?

Reply:
Yes, but technically we allow to use Detox with other runners as well, so... 🤷‍♂️